site/src/xServices/StateContext.tsx

```diff
@@ -35,6 +37,7 @@ export const XServiceProvider: FC<{ children: ReactNode }> = ({ children }) => {
         entitlementsXService: useInterpret(entitlementsMachine),
         siteRolesXService: useInterpret(siteRolesMachine),
         deploymentConfigXService: useInterpret(deploymentConfigMachine),
+        updateCheckXService: useInterpret(updateCheckMachine),
```

Comment:
I noticed some type errors in this file. Those should clear up if you run `yarn run typegen` from the`site` directory. This command generates xservice types.